cli/src/commands/migration/start.ts

```diff
@@ -0,0 +1,125 @@
+import { Args, Flags } from '@oclif/core';
+import { BaseCommand } from '../../base.js';
+import { getBranchDetailsWithPgRoll, isBranchPgRollEnabled } from '../../migrations/pgroll.js';
+import chalk from 'chalk';
+import path from 'path';
+import { safeJSONParse, safeReadFile } from '../../utils/files.js';
+
+export default class MigrationStart extends BaseCommand<typeof MigrationStart> {
+  static description = 'Start a new migration';
+
+  static examples = [];
+
+  static migrationJsonFlag = {
+    'migration-json': Flags.string({
+      helpValue: `
```

Comment:
TODO: need to check how this renders in CLI help, while currently hidden, I might have to turn this into a single line.

Reply:
Ah it was this that was causing the build CI to fail with

`Invalid count value`